enforcer-rules/src/main/java/org/apache/maven/plugins/enforcer/AbstractBanDependencies.java

```diff
@@ -63,22 +63,25 @@ public void execute( EnforcerRuleHelper helper )
             throw new EnforcerRuleException( "Unable to retrieve the MavenProject: ", eee );
         }
 
-        try
```

Comment:
Note that [most of the changes are just reindentation](https://github.com/apache/maven-enforcer/pull/32/files?w=1).